Modules/Core/Common/include/itkHexahedronCell.hxx

```diff
@@ -452,7 +452,7 @@ HexahedronCell<TCellInterface>::EvaluatePosition(CoordinateType *          x,
     else if ((itk::Math::abs(pcoords[0]) > ITK_DIVERGED) || (itk::Math::abs(pcoords[1]) > ITK_DIVERGED) ||
              (itk::Math::abs(pcoords[2]) > ITK_DIVERGED))
     {
-      return -1;
+      return false;
```

Comment:
This looks like a major bug fix, thanks! Did it not cause any test failures?

Would the return value of `EvaluatePosition` of both "Cell types" be important to any one? If so, maybe it should be separate bug report + pull request.

Reply:
I confess I did not dig deep.  I've just stuck an assert(0) in the 3 places the comment "(S.Hirschberg 11.12.2001)" occurs, and no test enters any of those 3 branches. So there is no test coverage for this change.

I probably should amend the commit message to say that behaviour is changing here, from returning true to returning false. Hopefully this swap is correct. I chose false because the 3rd place already has false.

I made this change as a separate commit, I could make a separate PR, but not sure what benefit that gives...

Reply:
OK I've dropped the commit.

Reply:
Thanks @seanm Sorry I don't want to discourage you. Finding bugs like this is really a major benefit from running code analysis tools, like you're doing. But then it's not always obvious how to deal with such bugs. Sometimes it's good enough to just make a note about them, without fixing them directly.  Ideally you would create a bug report, a unit test and a fix, but that's not always worth the effort 🤷